lib/node_modules/@stdlib/iter/cusome-by/README.MD

```diff
@@ -0,0 +1,238 @@
+Here's the adapted README.md for `cusome-by`, based on the `some-by` version:
+
+```markdown
```

Comment:
This is not how to write README files. Furthermore, the README name is incorrect. We don't uppercase `MD`.